eclass/linux-mod-r1.eclass

```diff
@@ -130,6 +126,14 @@ BDEPEND="
 IDEPEND="
 	sys-apps/kmod[tools]
 "
+if [[ ! ${LINUX_MOD_R1_NO_KERNEL_DEPS} ]]; then
+	RDEPEND+="
+		dist-kernel? ( virtual/dist-kernel:= )
```

Comment:
Won't IUSE="dist-kernel" exist for nothing here? Gonna need to `IUSE+=" dist-kernel"`

Reply:
Hrm, well it is used later on for a warning:
```
			if use dist-kernel; then
				ewarn "    <=virtual/dist-kernel-${max} or"
```
so should add `in_iuse dist-kernel &&` too

Do wish wouldn't have to worry about this, but better than a IUSE that does nothing.